onnx/defs/experiments/functions.cc

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "onnx/defs/function.h"
+using namespace ONNX_NAMESPACE;
+
+static Common::Status BuildMVN(std::unique_ptr<FunctionProto>* func_proto) {
+  if (nullptr == func_proto) {
+    return Status(
+        Common::CHECKER,
+        Common::INVALID_ARGUMENT,
+        "func_proto should not be nullptr.");
+  }
+
+  func_proto->reset(new FunctionProto);
+  auto& func = **func_proto;
+  func.set_name("FuncMeanVarianceNormalization");
+  func.set_doc_string(
+      "A MeanVarianceNormalization Function: Perform mean variance normalization "
+      "on the input tensor X using formula: <br/> ``` (X-EX)/sqrt(E(X-EX)^2) ``` <br/>"
+      "<b>INPUT: </b>X(float/float16/double) with Shape [N,C,W,H] <br/>"
+      "<b>ATTRIBUTE: </b><br/>&nbsp;&nbsp;&nbsp;&nbsp;axes: will be passed to ReducedMean Ops. "
+      "Use [0,2,3] for no across channel, [0,1,2,3] for across channel caculation.<br/>"
```

Comment:
for no across channel ---> for calculating means and variances along channels. Two variables with the same C-coordinate are associated with the same mean and variance.
for across channel ---> to calculate global mean and  global variance. In this case, all variables share the same mean/variance.